src/verifiers/issuerIdentity/dnsDid/dnsDidProof.test.ts

```diff
@@ -43,98 +53,114 @@ describe("test", () => {
+      expect(openAttestationDnsDidIdentityProof.test(v3DidSigned, options)).toBe(false);
+    });
+    it("should return false for document using `DNS-TXT` top level identity proof", () => {
+      expect(openAttestationDnsDidIdentityProof.test(v3DocumentStoreIssued, options)).toBe(false);
+      expect(openAttestationDnsDidIdentityProof.test(v3TokenRegistryIssued, options)).toBe(false);
+    });
+  });
 });
 
 describe("verify", () => {
-  it("should verify a document with dns binding to did", async () => {
-    const fragment = await openAttestationDnsDidIdentityProof.verify(documentDnsDidSigned, options);
-    expect(fragment).toMatchInlineSnapshot(`
-      Object {
-        "data": Array [
-          Object {
-            "key": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89#controller",
-            "location": "example.tradetrust.io",
-            "status": "VALID",
-          },
-        ],
-        "name": "OpenAttestationDnsDidIdentityProof",
-        "status": "VALID",
-        "type": "ISSUER_IDENTITY",
-      }
-    `);
-  });
-
-  it("should verify a document without dns binding to did", async () => {
-    const fragment = await openAttestationDnsDidIdentityProof.verify(documentDnsDidNoDnsTxt, options);
-    expect(fragment).toMatchInlineSnapshot(`
-      Object {
-        "data": Array [
-          Object {
-            "key": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89#controller",
-            "location": "example.com",
-            "status": "INVALID",
+  describe("v2", () => {
+    it("should verify a document with dns binding to did", async () => {
+      const fragment = await openAttestationDnsDidIdentityProof.verify(documentDnsDidSigned, options);
+      expect(fragment).toMatchInlineSnapshot(`
+        Object {
+          "data": Array [
+            Object {
+              "key": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89#controller",
+              "location": "example.tradetrust.io",
+              "status": "VALID",
+            },
+          ],
+          "name": "OpenAttestationDnsDidIdentityProof",
+          "status": "VALID",
+          "type": "ISSUER_IDENTITY",
+        }
+      `);
+    });
+    it("should verify a document without dns binding to did", async () => {
+      const fragment = await openAttestationDnsDidIdentityProof.verify(documentDnsDidNoDnsTxt, options);
+      expect(fragment).toMatchInlineSnapshot(`
+        Object {
+          "data": Array [
+            Object {
+              "key": "did:ethr:0xE712878f6E8d5d4F9e87E10DA604F9cB564C9a89#controller",
+              "location": "example.com",
+              "status": "INVALID",
+            },
+          ],
+          "name": "OpenAttestationDnsDidIdentityProof",
+          "status": "INVALID",
+          "type": "ISSUER_IDENTITY",
+        }
+      `);
+    });
+    it("should error if document has issuers not using DNS-DID", async () => {
```

Comment:
only test that changed, right now returning only error since the entire document isnt valid anyway (not suppose to mix)